geometry_msgs/Point point_1
geometry_msgs/Point point_2
